roles/usb_lib/tasks/install.yml

```
@@ -47,42 +47,62 @@
   when: udev_unit.stat.exists is defined and udev_unit.stat.exists
 
 # http://raspbian.raspberrypi.org/raspbian/pool/main/u/usbmount/usbmount_0.0.22_all.deb
-- name: Install {{ iiab_download_url }}/usbmount_0.0.22_all.deb, no longer supported by {RasPiOS, Debian, Ubuntu}
-  apt:
-    deb: "{{ iiab_download_url }}/usbmount_0.0.22_all.deb"
-  # when: is_debian
-
-# check status of usbmount on mintlinux - should be ok Ubuntu variant
-# - name: Install usbmount from OS repo for Ubuntu variants
-#   package:
-#     name: usbmount
-#     state: present
-#   when: is_ubuntu
-
-- name: Add dir {{ doc_root }}/local_content, where USB drive links can appear (0775)
+#- name: Install {{ iiab_download_url }}/usbmount_0.0.22_all.deb, no longer supported by {RasPiOS, Debian, Ubuntu}
+#  apt:
+#    deb: "{{ iiab_download_url }}/usbmount_0.0.22_all.deb"
+#  # when: is_debian
+
+- name: Install lockfile-progs util-linux for usbmount from OS repo
+  package:
+    name:
+    - lockfile-progs
+    - util-linux
```

1. Looks great.  @avni indicates she'll do a clean test (clean UNIT TEST) today, with:

`curl iiab.io/install.txt | bash -s 3916 3925`

```suggestion
      - lockfile-progs
      - util-linux
```

2. Ideally @tim-moody can answer her question to clarify what she should be testing and how:

- https://github.com/iiab/iiab/pull/3925#issuecomment-2624533223